### ISSUE TITLE
Very slow on Netbeans 8.1 

### ISSUE THREAD

**Issue by zrii** [guru-digital/sstemplate-netbeans#2]
The plugin works on it, but is very slow when formatting.

**Comment by cjsewell** [comment]
Yeah sorry. This is work in progress and there are still a lot to be done.

Feel free to make a pull request!
